layouts/partials/deprecation-warning.html

```diff
@@ -9,4 +9,13 @@ <h3>
     </p>
   </div>
 </section>
+{{ else if and (eq .Section "blog") .Date (.Date.Before (now.AddDate -1 0 0)) -}}
+{{ $title := .Param "outdated_blog.title" | default (T "outdated_blog_title") -}}
+{{ $msg := .Param "outdated_blog.message" | default (T "outdated_blog_over_one_year_old")  -}}
```

Comment:
I think that I'll pull out the `.Param "outdated_blog.*` parts, since it isn't fully functional and it isn't needed for marking blogs that are a year or older. (I'll do that tomorrow.)